src/backend/metal/src/conversions.rs

```diff
@@ -202,75 +878,91 @@ pub fn map_blend_op(operation: &pso::BlendOp) -> (MTLBlendOperation, MTLBlendFac
     use hal::pso::BlendOp::*;
 
     match *operation {
-        Add    { src, dst } => (MTLBlendOperation::Add,             map_factor(src), map_factor(dst)),
-        Sub    { src, dst } => (MTLBlendOperation::Subtract,        map_factor(src), map_factor(dst)),
-        RevSub { src, dst } => (MTLBlendOperation::ReverseSubtract, map_factor(src), map_factor(dst)),
-        Min => (MTLBlendOperation::Min, MTLBlendFactor::Zero, MTLBlendFactor::Zero),
-        Max => (MTLBlendOperation::Max, MTLBlendFactor::Zero, MTLBlendFactor::Zero),
+        Add { src, dst } => (MTLBlendOperation::Add, map_factor(src), map_factor(dst)),
+        Sub { src, dst } => (
+            MTLBlendOperation::Subtract,
+            map_factor(src),
+            map_factor(dst),
+        ),
+        RevSub { src, dst } => (
+            MTLBlendOperation::ReverseSubtract,
+            map_factor(src),
+            map_factor(dst),
+        ),
+        Min => (
+            MTLBlendOperation::Min,
+            MTLBlendFactor::Zero,
+            MTLBlendFactor::Zero,
+        ),
+        Max => (
+            MTLBlendOperation::Max,
+            MTLBlendFactor::Zero,
+            MTLBlendFactor::Zero,
+        ),
     }
 }
 
-
 pub fn map_vertex_format(format: Format) -> Option<MTLVertexFormat> {
-    use metal::MTLVertexFormat::*;
     use hal::format::Format as f;
-    Some(match format {  
-        f::R8Unorm     => UCharNormalized,
-        f::R8Inorm     => CharNormalized,
-        f::R8Uint      => UChar,
-        f::R8Int       => Char,
-        f::Rg8Unorm    => UChar2Normalized,
-        f::Rg8Inorm    => Char2Normalized,
-        f::Rg8Uint     => UChar2,
-        f::Rg8Int      => Char2,
-        f::Rgb8Unorm   => UChar3Normalized,
-        f::Rgb8Inorm   => Char3Normalized,
-        f::Rgb8Uint    => UChar3,
-        f::Rgb8Int     => Char3,
-        f::Rgba8Unorm  => UChar4Normalized,
-        f::Rgba8Inorm  => Char4Normalized,
-        f::Rgba8Uint   => UChar4,
-        f::Rgba8Int    => Char4,
-        f::Bgra8Unorm  => UChar4Normalized_BGRA,
-        f::R16Unorm    => UShortNormalized,
-        f::R16Inorm    => ShortNormalized,
-        f::R16Uint     => UShort,
-        f::R16Int      => Short,
-        f::R16Float    => Half,
-        f::Rg16Unorm   => UShort2Normalized,
-        f::Rg16Inorm   => Short2Normalized,
-        f::Rg16Uint    => UShort2,
-        f::Rg16Int     => Short2,
-        f::Rg16Float   => Half2,
-        f::Rgb16Unorm  => UShort3Normalized,
-        f::Rgb16Inorm  => Short3Normalized,
-        f::Rgb16Uint   => UShort3,
-        f::Rgb16Int    => Short3,
-        f::Rgb16Float  => Half3,
+    use metal::MTLVertexFormat::*;
+    Some(match format {
+        f::R8Unorm => UCharNormalized,
```

Comment:
please don't mix formatting changes with functional ones, unless it's too late by now :)